spec.bs

```diff
@@ -206,7 +207,7 @@ This limitation motivates the following Secure Payment Confirmation behavior:
 
 1. SPC supports cross-origin registration from an iframe in a third-party
     context. For instance, this registration might take place following some
-    other identity and verification (<abbr>ID&amp;V</abbr>) flow (e.g., SMS OTP).
+    other identity and verification (<abbr>ID and V</abbr>) flow (e.g., SMS OTP).
```

Comment:
Nit; did this need changed or could it be left as-is?

Reply:
I used ID&amp;V in the scope doc. I have no objection to leaving as-is here.